nav2_collision_monitor/include/nav2_collision_monitor/polygon.hpp

```diff
@@ -251,6 +264,10 @@ class Polygon
   ActionType action_type_;
   /// @brief Minimum number of data readings within a zone to trigger the action
   int min_points_;
+  /// @brief Minimum number of consecutive data with minimum min_points to trigger the action
+  int min_consecutive_in_;
```

Comment:
On naming: this a little ambiguous as a variable / parameter name. Can you come up with some alternatives?